Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -298,12 +296,24 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
 - (void)toxAV:(OCTToxAV *)toxAV audioBitRateChanged:(OCTToxAVAudioBitRate)bitrate stable:(BOOL)stable friendNumber:(OCTToxFriendNumber)friendNumber
 {
-    // Lower bitrate if unstable?
+    NSArray *validBitrates = @[@8, @16, @24, @32, @48];
+
+    if (! stable) {
```

Comment:
Early return would look nicer (at the start of method, before validBitrates array)
```
if (stable) {
    return;
}
```